mackup/constants.py

```diff
@@ -15,11 +17,14 @@
 # Default Mackup backup path where it stores its files in Dropbox
 MACKUP_BACKUP_PATH = 'Mackup'
 
-# Mackup config file
-MACKUP_CONFIG_FILE = '.mackup.cfg'
-
-# Directory that can contains user defined app configs
-CUSTOM_APPS_DIR = '.mackup'
+if os.environ.get('XDG_CONFIG_HOME'):
```

Comment:
Makes sense. I kinda felt that way too so I put it in a separate commit. Ill fix that.